subsys/nrf_security/cmake/nrf_config.cmake

```diff
@@ -107,6 +107,12 @@ if (NOT MBEDTLS_PSA_CRYPTO_SPM)
   kconfig_check_and_set_base(MBEDTLS_SSL_RENEGOTIATION)
   kconfig_check_and_set_base(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
   kconfig_check_and_set_base(MBEDTLS_SSL_PROTO_TLS1_2)
+  kconfig_check_and_set_base(MBEDTLS_SSL_PROTO_TLS1_3)
+  kconfig_check_and_set_base(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED)
+  kconfig_check_and_set_base(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_EPHEMERAL_ENABLED)
+  kconfig_check_and_set_base(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_EPHEMERAL_ENABLED)
+  kconfig_check_and_set_base(MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE)
+  kconfig_check_and_set_base(MBEDTLS_SSL_HANDSHAKE_WITH_PSK_ENABLED)
```

Comment:
Missing `MBEDTLS_TLS_SESSION_TICKETS`?

Reply:
Ah, missed it because I did not use it. `MBEDTLS_TLS_SESSION_TICKETS` is a config to define an already existing config: `MBEDTLS_SSL_TICKETS` the latter is already defined. So i'm unsure of what is the former used for. I will remove `MBEDTLS_TLS_SESSION_TICKETS` from `Kconfig.tls`